forch/forchestrator.py

```diff
@@ -87,10 +88,20 @@ def initialize(self):
         self._cpn_collector.initialize()
         LOGGER.info('Using peer controller %s', self._get_peer_controller_url())
 
-        self._faucetizer = Faucetizer()
-        devices_state = load_devices_state()
-        for mac, device_beahvior in devices_state.device_mac_behaviors:
-            self.process_device_behavior(mac, device_behavior)
+        structural_config_file = self.config.get('dva', {}).get(
+            'structural_config_file', _STRUCTURAL_CONFIG_DEFAULT)
+        structural_config_path = os.path.join(
+            os.getenv('FAUCET_CONFIG_DIR'), structural_config_file)
+        if self._config.get('dva', {}).get('run_faucetizer', False):
+            with open(structural_config_path) as file:
+                structural_config = yaml.safe_load(file)
+                self._faucetizer = faucetizer.Faucetizer(structural_config)
+
+        static_behaviors_file = self._config.get('dva', {}).get('static_device_behavior')
```

Comment:
dva is too specific since all this stuff, for example, also applies to roles and other device behavior considerations. Maybe put them under an "orchestration" bucket.

also, above you have the config dir prefixed by FAUCET_CONFIG_DIR env but not here?  Seems like both should have a similar treatment (even if with different env variables)

Also add logging that says "loading structural config from %s" -- so it's clear to people which actual file is being used.

Reply:
Done